package.json

```diff
@@ -59,6 +60,7 @@
     "morgan": "^1.7.0",
     "node-sass": "^3.12.4",
     "postcss-loader": "^1.1.1",
+    "precommit-hook": "^3.0.0",
```

Comment:
We should not do this - I switched to husky because it was easier to use. If you issues complaining about precommit-hook, you need to clear out your old hook in `.git/hooks/pre-commit`